### ISSUE TITLE
Doesn't work on directories with spaces (WinXP)

### ISSUE THREAD

**Issue by ksrik** [wting/autojump#418]
If the directory name has *spaces*, say like **My eBooks**, I can add this to the database. But I cannot jump to it.

It fails to find this directory if I give only a partial name.
e.g. This will not work:
`
j My eBooks
`
`
j My
`

Is this a known issue?

regards,
Srikk

**Comment by wting** [comment]
An simpler invocation would be `j ebooks`.

`j My eBooks` passes _two separate parameters_ to `autojump`, which is handled differently. What you probably want is `j "My eBooks"` to pass it as a single parameter.

Multiple parameter invocation matches one parameter per node. For example given `C:\Program Files\Adobe\data` you can jump that path via `j adobe data` or `j program adobe data`.

**Comment by ksrik** [comment]
Tried `j ebooks`

It does not work. I added the path again and tried, but it still fails.
Even `j "My eBooks"` does not work.

Error is `autojump: directory ebooks not found`.

I am using version 22.2.4.

Regards,
Srikk

**Comment by wting** [comment]
I've confirmed this when writing tests.

**Comment by ksrik** [comment]
Is any fix coming soon?

**Comment by wting** [comment]
No, as I am pre-occupied with other things. The problem is with how [regex is splitting up Windows paths](https://github.com/wting/autojump/blob/master/bin/autojump#L277).

Pull requests are more than welcome though.

**Comment by i-cant-git** [comment]
Please try now as #381 is merged
